tools/automator-main.sh

```diff
@@ -226,6 +231,15 @@ validate_opts() {
   if [ -z "${git_exclude:-}" ]; then
     git_exclude=""
   fi
+
+  if $fetch_tags; then
```

Comment:
this will fail if `--fetch-tags` is not passed in the command line. I suggest to initialize `fetch_tags=false` at the top the `validate_opts` function

Reply:
Adjusted.